client_restaurant/shared/src/commonMain/kotlin/data/local/mapper/Token.kt

```diff
@@ -0,0 +1,10 @@
+package data.local.mapper
+
+import data.remote.model.UserTokensDto
+import domain.entity.UserTokens
+
+
+fun UserTokensDto.toEntity() = UserTokens(
+    accessToken = accessToken,
+    refreshToken = refreshToken
+)
```

Comment:
Bareq noted that we don't need anymore a data class for UserTokens in the domain layer.

Reply:
Really?

Reply:
Yup